admin/nyx/Makefile

```diff
@@ -0,0 +1,37 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nyx
+PKG_VERSION:=2.1.0
+PKG_RELEASE:=1
+
+PYPI_NAME:=nyx
+PKG_HASH:=88521488d1c9052e457b9e66498a4acfaaa3adf3adc5a199892632f129a5390b
+
+PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
+PKG_LICENSE:=GPL-3.0-only
+PKG_LICENSE_FILES:=LICENSE
+
+include ../../lang/python/pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/python/python3-package.mk
+
+define Package/nyx
+  SECTION:=admin
+  CATEGORY:=Administration
+  URL:=https://nyx.torproject.org/
+  TITLE:=nyx
+  DEPENDS:=+python3-light +python3-stem
```

Comment:
python3-stem is not packaged in OpenWrt.
If you agree, I can send pull request for python3-steam from Turris OS packages repository
(https://gitlab.labs.nic.cz/turris/turris-os-packages/blob/develop/lang/python-stem/Makefile), but before submitting I need to improve it and change it to use pypi.

Reply:
My mistake. I had in on my tree but I did not include it in the PR. I've updated my local package to the latest git commit and included it in the PR.